HDDS-5912. Add description of hdds.datanode.dir in Getting Started (#2783)
apache/ozone: hadoop-hdds/docs/content/start/OnPrem.md

    </property>
{{< /highlight >}}

> * **hdds.datanode.dir** Determines where on the local filesystem HDDS data will be
> stored. Defaults to dfs.datanode.data.dir if not specified.
> 
> Here is an  example,
> {{< highlight xml >}}
>     <property>
>        <name>hdds.datanode.dir</name>
>        <value>/data/disk1/data</value>
>     </property>
> {{< /highlight >}}
> 

Reviewer: This section lists properties which are present in the `ozone-site.xml` config file created by `ozone genconf`.  `hdds.datanode.dir` is not such a property, so I think this section should not list it.

```suggestion
```

| ozone.scm.client.address       | SCM server name and port     | Used by client-side                                              |
| ozone.scm.datanode.address     | SCM server name and port     | Used by datanode to talk to SCM                                  |
| ozone.om.address               | OM server name               | Used by Ozone handler and Ozone file system.                     |

> | hdds.datanode.dir               | file path                | The HDDS data will be stored here.                     |

Reviewer: ```suggestion
| hdds.datanode.dir               | file path                | HDDS datanodes store data in this directory.           |
```
